lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -7,6 +7,8 @@
 
 /* eslint-env jest */
 
+jest.mock('../../lib/stack-collector.js', () => () => Promise.resolve([]));
```

Comment:
apparently `babel-jest` was hoisting these for us. The mock call just needs to happen before the mocked file is (transitively) required.